libcodechecker/parse.py

```diff
@@ -0,0 +1,175 @@
+
+        # Description is shown when the command's help is queried directly
+        'description': "Parse and pretty-print the summary and results from "
+                       "one or more 'codechecker-analyze' result files.",
+
+        # Help is shown when the "parent" CodeChecker command lists the
+        # individual subcommands.
+        'help': "Print analysis summary and results in a human-readable "
+                "format."
+    }
+
+
+def add_arguments_to_parser(parser):
+    """
+    Add the subcommand's arguments to the given argparse.ArgumentParser.
+    """
+
+    parser.add_argument('input',
+                        type=str,
+                        nargs='*',
+                        metavar='file/folder',
+                        default=os.path.join(util.get_default_workspace(),
+                                             'reports'),
+                        help="The analysis result files and/or folders "
+                             "containing analysis results which should be "
+                             "parsed and printed.")
+
+    parser.add_argument('-t', '--type', '--input-format',
+                        dest="input_format",
+                        required=False,
+                        choices=['plist'],
+                        default='plist',
+                        help="Specify the format the analysis results were "
+                             "created as.")
+
+    parser.add_argument('--suppress',
+                        type=str,
+                        dest="suppress",
+                        default=argparse.SUPPRESS,
+                        required=False,
+                        help="Path of the suppress file to use. Records in "
+                             "the suppress file are used to suppress the "
+                             "display of certain results when parsing the "
+                             "analyses' report. (Reports to an analysis "
+                             "result can also be suppressed in the source "
+                             "code -- please consult the manual on how to "
+                             "do so.) NOTE: The suppress file relies on the "
+                             "\"bug identifier\" generated by the analyzers "
+                             "which is experimental, take care when relying "
+                             "on it.")
+
+    parser.add_argument('--print-steps',
+                        dest="print_steps",
+                        action="store_true",
+                        required=False,
+                        help="Print the steps the analyzers took in finding "
+                             "the reported defect.")
+
+    add_verbose_arguments(parser)
+    parser.set_defaults(func=main)
+
+
+def parse(f, context, metadata_dict, suppress_handler, steps):
+    """
+    Prints the results in the given file to the standard output in a human-
+    readable format.
+    """
+
+    if not f.endswith(".plist"):
```

Comment:
I think we should inform the user that only files with `.plist` extension will be parsed.